README.md

```diff
@@ -106,6 +106,35 @@ Money::Currency.new("JPY").minor_units  # => 0
 Money::Currency.new("MGA").minor_units  # => 1
 ```
 
+## Storing money
+
+Since money internally uses BigDecimal it's logical to use a `decimal` column 
+for your database. The following examples are for ActiveRecord:
+
+```ruby
+create_table :orders do |t|
+  t.string :currency, limit: 3
+  t.decimal :total_price, precision: 20, scale: 3, null: false, default: '0.000'
+end 
+``` 
+
+While `money_accessor` can build you some accessor getters and setters, we 
+recommend to use the right APIs instead as they enable more expressive
+interactions and you get caching for free:
+
+```ruby
+class Order < ApplicationRecord
+  composed_of :total_price, 
+    class_name: 'Money', 
+    mapping: [%w(price value), %w(currency currency)]
```

Comment:
The mapping doesn't look to line up with the table you defined above in your example, shouldn't this be `total_price` instead of `price`? Not sure if the naming is to avoid conflict with the attribute names too